charts/unspec-service/values.preview.template.yaml

```diff
@@ -1,12 +1,45 @@
 tags:
   unspec-ccd-stack: true
 
+postgresql:
+  enabled: true
+  image:
+    tag: '11.3.0'
+  resources:
+    requests:
+      cpu: 25m
+  initdbScripts:
+    init.sql: |-
+      CREATE DATABASE cmc WITH OWNER = hmcts ENCODING = 'UTF-8' CONNECTION LIMIT = -1;
+      CREATE DATABASE "data-store" WITH OWNER = hmcts ENCODING = 'UTF-8' CONNECTION LIMIT = -1;
+      CREATE DATABASE "definition-store" WITH OWNER = hmcts ENCODING = 'UTF-8' CONNECTION LIMIT = -1;
+      CREATE DATABASE "user-profile" WITH OWNER = hmcts ENCODING = 'UTF-8' CONNECTION LIMIT = -1;
+  postgresqlUsername: hmcts
+  postgresqlPassword: hmcts
+  persistence:
+    enabled: false
+
 java:
   image: ${IMAGE_NAME}
   ingressHost: ${SERVICE_FQDN}
+  keyVaults:
+    unspec:
+      resourceGroup: unspec
+      secrets:
+        - unspec-idam-client-secret
+        - microservicekey-unspec-service
+        - system-update-user-username
+        - system-update-user-password
+        - AppInsightsInstrumentationKey
+        - docmosis-api-key
```

Comment:
why? this is defined in values.yaml

Reply:
we are overriding the keyvaults from values.yaml other wise it will pick db password which will be different then what it is created with in preview

Reply:
I thought that values.yaml picked secrets from the relevant ketvault based on what env it was deployed to? So this code is just duplicate?

Reply:
@swalker125 I think that the case here is not to pick `cmc-db-password` secret for preview env.

Reply:
@dharmendrak If we provide `CMC_DB_PASSWORD` in env variables, shouldn't it overwrite values picked from the vault?